aptos-move/aptos-vm-types/src/storage.rs

```diff
@@ -278,6 +278,7 @@ impl StoragePricing {
 #[derive(Clone, Debug)]
 pub struct ChangeSetConfigs {
     gas_feature_version: u64,
+    max_write_ops_per_transaction: u64,
```

Comment:
can we move it to the end to align with below?

Reply:
okay...